### ISSUE TITLE
[question] Choice of using FLANN

### ISSUE THREAD

**Issue by fferroni** [IntelVCL/Open3D-PointNet2-Semantic3D#31]
Hello,

Great work on Open3D version 0.5.0
I was just wondering what your motivation was to use FLANN rather than use i.e. an octree data-structure to run the ThreeNN operations directly on the GPU, which has been shown in many contexts to run faster? Was it simply the ease of using already an available library, or something to do with performance?

Best!

**Comment by yxlao** [comment]
You're right that if we use CUDA KNN libraries, this operation may run faster. We're currently using CPU implementation is because we have a convenient implementation in Open3D. In the future, we're planning GPU acceleration to Open3D, which could further accelerate this implementation.
